pkgs/applications/misc/tandoor-recipes/default.nix

```diff
@@ -0,0 +1,143 @@
+{ lib
+, fetchFromGitHub
+, fetchpatch
+, callPackage
+, python3
+}:
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      django = super.django_4;
+
+      # Tests are incompatible with Django 4
+      django-js-reverse = super.django-js-reverse.overridePythonAttrs (_: {
+        doCheck = false;
+      });
```

Comment:
See https://github.com/ierror/django-js-reverse/pull/114 for when this isn't needed anymore.